### DISCUSSION TITLE
Hex VTK Parallel Bug

### DISCUSSION THREAD

**Discussion by makeclean** [idaholab/moose#22273] [Q&A General]
Hi All

I think there is a bug from libmesh, when running an hex problem in parallel there are oddies along the domain processor boundaries, e.g.

![image](https://user-images.githubusercontent.com/2439754/193270475-d1222e9d-de32-4cf0-b1af-e364c46ea952.png)

Am I doing something odd, or is this a known problem? Key thing, this is using VTK out

Thanks

Andy

**Comment by GiudGiud** [comment]
@roystgnr has the VTK output been set up for IGA?

**Reply by makeclean** [reply]
Sorry I originally tagged this as an IGA mesh, it isn't its a hex mesh

**Reply by GiudGiud** [reply]
Is the exodus output normal btw?
And what kind of Hex is this?

**Reply by roystgnr** [reply]
It's been set up; not sure how reliable it is.  CoreForm wrote the VTK+IGA code, we got it all merged in libMesh and MOOSE including VTK-based XML diff tests in https://github.com/idaholab/moose/pull/20783 ... and then we started seeing diffs on older build boxes upgrading to VTK 9.0.  I thought VTK 9.1 was safe, but digging back into it I don't see any response from @loganharbour to my question in https://github.com/idaholab/moose/issues/21449 and I'm not liking the strikethrough in the comment in https://github.com/idaholab/moose/pull/21450 ...

**Comment by friedmud** [comment]
Hey Andy ( @makeclean ), we need some more info here.  Is this only with VTK - or does this happen with Exodus output as well?  Are you using a pre-split mesh?  Can you post the mesh file somewhere?

**Reply by makeclean** [reply]
I've not tried with exodus, I'll try and report back. It is not pre-split, I can post inputs & mesh no problem.

**Reply by makeclean** [reply]
Exodus is fine, see below;
![Screenshot from 2022-10-04 21-29-53](https://user-images.githubusercontent.com/2439754/193920600-7e792a03-a179-49fe-95df-be9d84ce6bd0.png)
LHS is the temp variable with errors, middle is the partition map and RHS is the exodus file. 

The meshes are too large to trivially attach - https://file.io/toEiznczSbNI

**Comment by GiudGiud** [comment]
Any update on this?
@friedmud did you get the chance to check out the mesh?
